pkg/kubelet/cm/dra/state/state_checkpoint_test.go

```diff
@@ -50,21 +50,23 @@ func TestCheckpointGetOrCreate(t *testing.T) {
 		},
 		{
 			"Restore valid checkpoint",
-			`{"version":"v1","entries":[{"DriverName":"test-driver.cdi.k8s.io","ClaimUID":"067798be-454e-4be4-9047-1aa06aea63f7","ClaimName":"example","Namespace":"default","PodUIDs":{"139cdb46-f989-4f17-9561-ca10cfb509a6":{}},"CDIDevices":["example.com/example=cdi-example"]}],"checksum":2242470059}`,
+			`{"version":"v1","entries":[{"DriverName":"test-driver.cdi.k8s.io","ClaimUID":"067798be-454e-4be4-9047-1aa06aea63f7","ClaimName":"example","Namespace":"default","PodUIDs":{"139cdb46-f989-4f17-9561-ca10cfb509a6":{}},"CDIDevices":{"test-driver.cdi.k8s.io":["example.com/test=cdi-test"]}}],"checksum":170457543}`,
 			"",
```

Comment:
not that important but you change example.com/example=cdi-example to example.com/test=cdi-test

Reply:
Will change back -- was a merge-conflict issue since I had this implemented before you updated your previous PR to remove the nvidia references.

Reply:
Updated.